gradle.properties

```diff
@@ -1,6 +1,6 @@
 kotlinVersion=1.0.6
 junitPlatformVersion=1.0.0-M3
-kotlin.incremental=true
+kotlin.incremental=false
```

Comment:
Forgot to remove that. Compiler was acting wonky, my default behaviour is to disable incremental compilation :).